Sources/PointFree/Episode/Episode.swift

```diff
@@ -101,11 +103,16 @@ private let topLevelTagsView = View<[Tag]> { tags in
     ])
 }
 
-private let episodeTocView = View<[Episode.TranscriptBlock]> { blocks in
+private let episodeTocView = View<[TranscriptBlock]> { blocks in
   blocks
-    .filter { $0.type == .title && $0.timestamp != nil }
-    .flatMap { block in
-      tocEntryView.view((block.content, block.timestamp ?? 0))
+    .flatMap { block -> [Node] in
+      switch block.type {
+      case let .title(timestamp):
```

Comment:
I moved the `timestamp` value into the `BlockType` instead of the `TranscriptionBlock`. the idea being not every block as a timestamp that makes sense (like images and code snippets), only certain blocks.